doc/api.js

```diff
@@ -147,11 +147,35 @@
  */
 
 /**
+ * Represents the device information of the debug probe
+ *
+ * @typedef ProbeInformation
+ *
+ * @property {integer} serialNumber The serialnumber of the probe
+ * @property {integer} clockSpeedkHz The programming speed
```

Comment:
"Programming speed" is a slightly inaccurate description I believe. How about something like "The clock speed of the probe interface"?